README.md

```diff
@@ -73,15 +73,37 @@ After the install process is over, you'll be able to run your application using
 $ grunt
 ```
 
-Your application should run on port 3000, so in your browser just go to [http://localhost:3000](http://localhost:3000)
+Your application should run on port 3000 with the *development* environment configuration, so in your browser just go to [http://localhost:3000](http://localhost:3000)
 
 That's it! Your application should be running. To proceed with your development, check the other sections in this documentation.
 If you encounter any problems, try the Troubleshooting section.
 
+* explore ```config/env/development.js``` for development environment configuration options
+
+### Running in Production mode
+To run your application with *production* environment configuration, execute grunt as follows:
+```bash
+$ grunt prod
+```
+* explore ```config/env/production.js``` for production environment configuration options
+
+## Running in a Secure environment
```

Comment:
Indeed I'lll change the 2 "#" to 3 so that it is after the 'running in production mode' and not within it (I think it's much clearer that way instead of nesting it too much levels).

Will also update the title.